src/enclave/rpc_context.h

```diff
@@ -103,7 +103,8 @@ namespace enclave
       set_response_header(name, fmt::format("{}", n));
     }
 
-    virtual bool response_is_error() const = 0;
+    virtual void set_apply_writes(bool apply) = 0;
```

Comment:
I wonder if an enum would be more descriptive:

```
enum Apply {
  ON_SUCCESS,
  ALWAYS
}
...
args.rpc_ctx->set_writes(Apply::ALWAYS);
```

But also, this is a perfectly good bike shed, and I will approve and shut up now.